cypress/commands/thermalcamera.js

```diff
@@ -1,36 +0,0 @@
-const url = require("url");
```

Comment:
this was using the real camera code running in docker.   So we probably want to leave it and get that working again at a later point.